samcli/local/docker/lambda_image.py

```diff
@@ -287,3 +289,18 @@ def _generate_dockerfile(base_image, layers):
         for layer in layers:
             dockerfile_content = dockerfile_content + f"ADD {layer.name} {LambdaImage._LAYERS_DIR}\n"
         return dockerfile_content
+
+    def _remove_images(self, repo_name: str) -> None:
+        """
+        Remove all images for given repo
+
+        Parameters
+        ----------
+        repo_name str
+            Repo for which all images will be removed
+        """
+        try:
+            for image in self.docker_client.images.list(name=repo_name):
+                self.docker_client.images.remove(image.id)
```

Comment:
This fails if there are other tags referencing this image (if there are multiple tags with the same image id). Should I use the `force` flag or should we never have multiple tags referencing the same image?
https://docs.docker.com/engine/reference/commandline/rmi/#examples